logwarn_openqa

```diff
@@ -78,6 +78,8 @@ $logwarn ${options} -m '\[.*:?(debug|info|warn|error)\]' -p ${file} \
     '!\[warn\].* Unable to wakeup scheduler: Request timeout' \
     `# https://progress.opensuse.org/issues/105918` \
     '!\[warn\].* fatal: Invalid revision range .*\.\.' \
+    `# https://progress.opensuse.org/issues/113030` \
+    `!\[warn\].* fatal: ambiguous argument \'\(unreadable git hash\)\.\.\(unreadable git hash\)\': unknown revision or path not in the working tree.` \
```

Comment:
please generalize to cover `\(unreadable git hash\)\.\.` as we have cases where the later job has a hash but the former doesn't

Reply:
Seems like the is-ignored unit test is broken. I added the other case and it's passing w/o further changes 🤔 Need to investigate this more

Reply:
Even `is-ignored '\[error\].+fatal: ambiguous argument e737020f1b7d8ff3f6cc3bbc009f83a4a09862cb'` also passes without further changes

Reply:
Aha! Apparently the wrong quotation marks cause the problem

Reply:
Solved with @perlpunk's help